addOns/ascanrulesAlpha/ascanrulesAlpha.gradle.kts

```diff
@@ -8,9 +8,18 @@ zapAddOn {
     manifest {
         author.set("ZAP Dev Team")
         url.set("https://github.com/zaproxy/zap-extensions/wiki/HelpAddonsAscanrulesAlphaAscanalpha")
+
+        dependencies {
+            addOns {
+                register("custompayloads")
```

Comment:
IMO this should be an optional dependency, I should be able to use the scanners without the Custom Payloads add-on (they work without it being installed).

Reply:
I'm good with this being an optional dependency, and I think the way the TestUserAgent example is setup it doesn't require the addon to be there (it adds the payloads if the addon is there, but there are still local defaults).

Reply:
I believe this is now addressed.